wordpress-mobile/WordPress-iOS
# Jetpack: display atomic sites

This PR makes Atomic sites be displayed in the Jetpack app.

### To test

1. Login with an account that has any Atomic site
2. Make sure the Atomic site appears on the list

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
